oap-ape/README.md

```diff
@@ -72,7 +72,7 @@ mvn clean install
 Native and Java build. We will use `$OAP_ROOT_DIR/oap-ape/ape-java/ape-spark/target/ape-spark-1.1.0-SNAPSHOT.jar`, `$OAP_ROOT_DIR/oap-ape/ape-java/ape-flink/ape-flink-x.xx.x/target/ape-flink-x.xx.x-1.1.0-SNAPSHOT.jar` later.
 ```
 cd $OAP_ROOT_DIR/oap-ape/ape-java
-mvn clean package -am -Pshading
+mvn clean package -am -PsparkNative
```

Comment:
`-PapeNative` instead?

Reply:
If we enable this option, which means dependency libraries (like arrow/ Parquet) must be static libraries, right? should we point it out in the document?

Reply:
yes, it's right.

Reply:
oh, now we all use static libraries when build ape-native even we don't use this option.

Reply:
have we changed dependency build scripts? like this https://github.com/oap-project/sql-ds-cache/blob/629289787b8e70f09393fe96accfed3ebe5fddef/oap-ape/install.sh#L61 
and I suggest add libredis.so step and other dependency in this file (make them compilation option is also a good idea, but need change some cpp source code)

Reply:
we have changed the scripts here https://github.com/oap-project/sql-ds-cache/blob/629289787b8e70f09393fe96accfed3ebe5fddef/ci_scripts/install_arrow.sh#L44

Reply:
when we will use  this script ,sql-ds-cache/oap-ape/install.sh ? and the arrow repo in this script is not the same with the repo in the install_arrow.sh.

Reply:
emmm this is for user I think. Seems we didn't update it.